docs/GTD_IMPLEMENTATION.md

```
@@ -56,6 +56,16 @@ Your core values and life mission that drive everything else:
 
 Each horizon file includes comprehensive templates to guide your planning and reflection.
 
+### Horizon Overview Pages
+
+Every Horizons folder now includes an instructive `README.md` built from the canonical template in `docs/horizon-readme-template.md`. The overview file:
+
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)